frontend/simple/js/event-log.js

```diff
@@ -0,0 +1,23 @@
+import {default as idbLog, attachVuex} from './idb-flume-log'
+import filter from './flumeview-filter'
+import map from './flumeview-map'
+import flumeDB from 'flumedb'
+var log
+var db
+export const loaded = idbLog().then((value) => { log = value })
+export const attachStore = attachVuex
+
+export default function EventLog () {
+  if (!db) {
+    db = flumeDB(log)
+    db.use('payment', filter((event) => {
+      return event.value.type === 'Payment'
+    }))
+    db.use('events', map((event) => {
+      return event
+    }))
+    return db
```

Comment:
So all of this stuff should be in the callback on line 7, not here.

If `EventLog` is an abstraction over flume, then that's what it should be. Right now you can just `export db as default` if you're holding off on implementing the abstraction.